Sources/CoreMetrics/Metrics.swift

```diff
@@ -12,220 +12,230 @@
 //
 //===----------------------------------------------------------------------===//
 
-public protocol Counter: AnyObject {
+public protocol CounterHandler: AnyObject {
```

Comment:
Document that these types are meant to be implemented by metrics libraries and users should only really interact with `Counter`?

Then the name of this one becomes less important to end users, and we can stick to Handler since that is aligned with LogHandler how it is in the logging

Reply:
yes will document more as we transition to the formal proposal. but agreed: 1/ this is not important to (since hidden from) end users, and 2/ should be consistent with logging api